src/miner.erl

```diff
@@ -575,6 +577,14 @@ create_block(Metadata, Txns, HBBFTRound, Chain, VotesNeeded, {MyPubKey, SignFun}
     HeightNext = HeightCurr + 1,
     Ledger = blockchain:ledger(Chain),
     SnapshotHash = snapshot_hash(Ledger, HeightNext, Metadata, VotesNeeded),
+    {ok, CurrentBlockHash} = blockchain:head_hash(Chain),
```

Comment:
pull this inside the validator clause?

Reply:
actually this line looks like it can be removed all together as the var is already created with a value derived from the block_info_v2

Reply:
Removed as part of refactoring out of the poc keys